doc/build-openbsd.md

```diff
@@ -92,22 +49,6 @@ cd db-4.8.30.NC/build_unix/
 make install # do NOT use -jX, this is broken
 ```
 
-### Resource limits
```

Comment:
I think we should keep this around just in case people run against these limits.

Reply:
Restored.